Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:10-alpine3.10
+
+WORKDIR /opt/referee
+
+# Other dependencies
+RUN apk add python make bash
+
+COPY package.json yarn.lock lerna.json ./
+
+RUN yarn
```

Comment:
I think running `yarn` without parameter triggers a `yarn install` so its redundant with the line below. I would remove this line and leave the explicit `yarn install`

Reply:
More info [here](https://classic.yarnpkg.com/en/docs/cli/#toc-default-command)

Reply:
Yeah, you're right. Removed.